CMakeLists.txt

```diff
@@ -92,6 +92,11 @@ else (WIN32)
     if (AWK STREQUAL "AWK-NOTFOUND")
         message(FATAL_ERROR "AWK not found")
     endif()
+
+    # Try to locate the paxctl tool. Failure to find it is not fatal,
```

Comment:
Do we need something to ensure that the tool is installed on the official build machines so that we do not release runtime without this bit set by accident?

Reply:
@jkotas I think we should add a distro with PAX enabled to our testing matrix.